### PR TITLE
NUTCH-2218 - Update CrawlComplete util with Commons CLI arg parsing

### PR DESCRIPTION
**Pull request by MJJoyce**
- Switch all argument parsing and checking to commons CLI.
- Update input directory processing such that the 'crawldb' folder
  should no longer be included in the input paths. Now the user simply
  points to the crawl folder instead of specific crawldb directories.

### CODE REVIEW

File: src/java/org/apache/nutch/util/CrawlCompletionStats.java [hunk 1]
```diff
@@ -61,27 +69,60 @@
   private int mode = 0;
 
   public int run(String[] args) throws Exception {
-    if (args.length < 3) {
-      System.err.println("Usage: CrawlCompletionStats inputDirs outDir mode [numOfReducer]");
+    Option helpOpt = new Option("h", "help", false, "Show this message");
+    Option inDirs = OptionBuilder
+        .withArgName("inputDirs")
+        .isRequired()
+        .withDescription("Comma separated list of crawl directories")
```
Comment (lewismc) [inline src/java/org/apache/nutch/util/CrawlCompletionStats.java]: This could do with being explicit e.g. providing an example

**Comment by lewismc** [comment]
Nice. +1 to this if you can provide an example.

**Comment by MJJoyce** [comment]
+1 @lewismc I'll see about throwing an example in there when I commit.

**Comment by MJJoyce** [comment]
Note, closing this manually since I buggered the merge =D

**Pull request closed without merging**